README.md

```diff
@@ -88,27 +90,47 @@ custom:
         request: # request mapping template name
         response: # response mapping template name
     schema: # defaults schema.graphql
-    serviceRole: "AppSyncServiceRole" # AppSyncServiceRole is a role defined by amazon and available in all accounts
```

Comment:
is this not required now?

Reply:
@sid88in This was not required and not even used since a long time ago.
This was moved into individual dataSources `serviceRoleArn` here https://github.com/sid88in/serverless-appsync-plugin/commit/0c113bb358f98ce73998f1c85e357121515b3748#diff-168726dbe96b3ce427e7fedce31bb0bcR102